### ISSUE TITLE
Problems to sync emotes

### ISSUE THREAD

**Issue by sidossauro** [cmooref17/Lethal-Company-TooManyEmotes#69]
First time reporting... from version 1.9.0 I noticed that the emotes are no longer synchronizing, it makes the same emote but at different times.

**Comment by cmooref17** [comment]
Yes, I am aware, but I forgot that I still had to address this. Thanks for reminding me! I will try to fix this in the next update. I will keep this ticket open as a reminder, thanks!

**Comment by MostlyPride** [comment]
Yo how am I suppose to do my banger emote songs if we can't sync.

This is the worst thing to happen since the holocaust :(

**Comment by MostlyPride** [comment]
![image](https://github.com/cmooref17/Lethal-Company-TooManyEmotes/assets/10325931/505e696c-636a-45cc-87eb-10a05e02f08b)

I am still sad.

Now here me out what if I get your a coffee

**Comment by cmooref17** [comment]
I don't need a coffee haha. I ran into some issues that I need to fix before I can push the update out. It was delayed :(
